.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Before you get started
+- [ ] 🙋‍♀️ Create an issue to discuss what you are doing to do!
+
+---
+
+## Description
+- Closes #issue_number
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 
+
+## Type of change
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How has this been tested?
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test configuration**:
+* Operating system:
+* Other relevant information:
+
+## Checklist
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
"Any changed dependencies have been added or removed correctly" is easier to understand (like in ESMValTool checklist) than downstream modules. In particular, this is related to package dependencies and required changes in environment_xxx.yaml and setup.py